src/cms/templates/pages/page_form.html

```diff
@@ -75,7 +75,7 @@ <h2 class="heading font-normal">
             {% else %}
                 <input type="submit" name="submit_review" class="cursor-pointer bg-blue-500 hover:bg-green-600 text-white font-bold py-3 px-4 rounded mr-2" value="{% trans 'Submit for review' %}" />
             {% endif %}
-            <button type="button" onclick="short_url_to_clipboard('s/p/{{ page_translation_form.instance.id }}');" class="cursor-pointer bg-blue-500 hover:bg-blue-600 text-white font-bold py-3 px-4 rounded">
+            <button type="button" onclick="short_url_to_clipboard('{{ page_translation.short_url }}');" class="cursor-pointer bg-blue-500 hover:bg-blue-600 text-white font-bold py-3 px-4 rounded">
```

Comment:
```suggestion
            <button type="button" onclick="short_url_to_clipboard('{{ page_translation_form.instance.short_url }}');" class="cursor-pointer bg-blue-500 hover:bg-blue-600 text-white font-bold py-3 px-4 rounded">
```
I'm sorry, I made a wrong suggestion here. It has to be `page_translation_form.instance` instead of `page_translation` because in the page form, the translation is referenced via the form variable...